wrapper.lisp

```diff
@@ -82,3 +82,6 @@
      (or speed (handle-max-speed handle))
      (* 1000000 (or delay 0))
      (or bits/word (handle-bits/word handle)))))
+
+(defun flush (handle)
```

Comment:
Instead of an explicit flush I think the wrapper should automatically call force-output on the handle after a write. That seems much closer to the expected behaviour.